src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -93,6 +93,7 @@ public static class ScriptConstants
         public const string AntaresColdStartHeaderName = "X-MS-COLDSTART";
         public const string SiteTokenHeaderName = "x-ms-site-restricted-token";
         public const string EasyAuthIdentityHeader = "x-ms-client-principal";
+        public const string BackgroundSyncHeader = "x-ms-client-backgroundsync";
```

Comment:
can we use something like to avoid potential conflicts
x-ms-functions-client-backgroundsync

Reply:
Thanks. Done updated.